examples/xed-asmparse-main.c

```diff
@@ -634,8 +634,8 @@ int main(int argc, char** argv)
     xed_enc_line_parsed_t* v = asp_get_xed_enc_node();
 
     process_args(argc, argv, v, &verbose);
-    asp_parse_line(v);
```

Comment:
I committed my own version of this one. I noticed when hunting down a bug you mentioned in email. sorry